src/main/java/org/breedinginsight/model/brapi_import/BrAPIImportConfigManager.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.breedinginsight.model.brapi_import;
+
+import io.micronaut.context.annotation.Context;
+import org.breedinginsight.model.brapi_import.config.*;
+import org.breedinginsight.model.brapi_import.imports.ImportMetadata;
+import org.breedinginsight.model.brapi_import.response.ImportConfig;
+import org.breedinginsight.model.brapi_import.response.ImportFieldConfig;
+import org.breedinginsight.model.brapi_import.response.ImportObjectConfig;
+import org.breedinginsight.model.brapi_import.response.ImportRelationOptionConfig;
+import org.reflections.Reflections;
+
+import javax.inject.Singleton;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Context
+@Singleton
+public class BrAPIImportConfigManager {
+
+    private Set<Class<?>> brAPIImports;
+
+    BrAPIImportConfigManager() {
+        // Get all imports
+        Reflections reflections = new Reflections("org.breedinginsight");
```

Comment:
Did you look at 3.16 Bean Annotation Metadata in the micronaut guide?

Reply:
I did, but I can't remember if I tried it or not. I'll take another look.